modules/datadict/php/datadict.class.inc

```diff
@@ -232,8 +243,8 @@ class Datadict extends \NDB_Menu_Filter
         return array_merge(
             $deps,
             array(
-             $baseURL . "/datadict/js/columnFormatter.js",
              $baseURL . "/datadict/js/datadict_helper.js",
```

Comment:
i think `datadict_helper.js` can be deleted since editing the description seems to be handled by columnFormatter